### ISSUE TITLE
Depth-based CNN missing from output folder

### ISSUE THREAD

**Issue by HKhawaja** [shicaiwei123/patch_based_cnn#6]
Awesome work on the implementation!! I saw that "depth_fasd.pth" is missing from the output/models folder. It is referenced in the test script: https://github.com/shicaiwei123/patch_based_cnn/blob/master/test/depth_cnn_test.py

**Comment by HKhawaja** [comment]
@shicaiwei123

**Comment by shicaiwei123** [comment]
Hello, sorry for the late reply.
The model depth_patch.pth has been uploaded just now.
It's worth noting that, here, only the depth map estimation is implemented and the SVM has not been added. The current strategy used to classify is to calculate the mean value of the estimated depth map and compare it with the threshold (here is 0.5), if it is bigger than the threshold, it is classified as the true face, otherwise, it is the false face.

**Comment by showbit01** [comment]
@shicaiwei123 is it only depth model or fusion of depth patch?

**Comment by shicaiwei123** [comment]
The performance post on the github only leverage the patch.

But we also provide the code for depth model



At 2022-05-20 14:36:36, "Shobhit Sharma" ***@***.***> wrote:

@shicaiwei123 is it only depth model or fusion of depth patch?

**Comment by showbit01** [comment]
Yeah have implemented the depth with help of prnet module but again that is not surprising to see the overfitting results..

**Comment by Ruhshan** [comment]
> Yeah have implemented the depth with help of prnet module but again that is not surprising to see the overfitting results..

Hi, can u help me in acquiring depth map with prnet? I tried this: 

https://github.com/shicaiwei123/patch_based_cnn/issues/13

Not sure it's the correct way or not.

Thanks in advance.

**Comment by showbit01** [comment]
> > Yeah have implemented the depth with help of prnet module but again that is not surprising to see the overfitting results..
> 
> Hi, can u help me in acquiring depth map with prnet? I tried this: 
> 
> https://github.com/shicaiwei123/patch_based_cnn/issues/13
> 
> Not sure it's the correct way or not.
> 
> Thanks in advance.

https://github.com/clks-wzz/PRNet-Depth-Generation

use this
